drivers/network/dd/e1000/ndis.c

```diff
@@ -222,7 +216,7 @@ MiniportInitialize(
 
     /* Allocate the DMA resources */
     Status = NdisMInitializeScatterGatherDma(MiniportAdapterHandle,
-                                             FALSE, // 32bit
+                                             TRUE, // 64bit is supported
```

Comment:
Does the e1000 support 64bit dma?

Reply:
Based on these google results, it seems 64bit DMA is questionable. Notice all the results mentioning how they had to disable 64-bit DMA to avoid Tx hangs:

![image](https://user-images.githubusercontent.com/685851/51502239-7b934780-1dd5-11e9-9e99-48d331111267.png)

Reply:
Am I right: restricting DMA to 32-bits only means that the kernel will have to allocate memory only within first 4GB of physical space?

Reply:
Yes. It's mostly a question of whether the device will accept physical addresses beyond 4GB (i.e. the size of a few particular registers). However if some devices are buggy, it of course wouldn't work despite the registers being 64 bit.